scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/model/SchedulingActivity.java

```diff
@@ -86,12 +88,13 @@ public static SchedulingActivity of(
         topParent,
         anchorId,
         anchoredToStart,
-        isNew
+        isNew,
+        null
     );
   }
 
   public SchedulingActivity withNewDuration(Duration duration){
-    return SchedulingActivity.of(
+    return new SchedulingActivity(
```

Comment:
Previous question applies here too, if naming an activity is common enough, maybe we want to overload `SchedulingActivity.of` instead of forcing the use of `new SchedulingActivity`, if that is the main pattern in the scheduler code.

Reply:
I found the many overloads of `SchedulingActivity.of` to be impossible to understand and keep track of, which is why I've deleted most of them. Since java doesn't have keyword args, and many of the arguments are nullable, its hard to tell what each argument is. I definitely don't want to add another overload, even if it was common.